pkg/apis/sriovnetwork/v1/helper.go

```diff
@@ -26,26 +26,74 @@ const (
 
 const invalidVfIndex = -1
 
-var SriovPfVfMap = map[string](string){
-	"1583": "154c",
-	"158b": "154c",
-	"10fb": "10ed",
-	"1015": "1016",
-	"1017": "1018",
+var log = logf.Log.WithName("sriovnetwork")
+var VfIds = []string{}
+
+// NicIdMap contains supported mapping of IDs with each in the format of:
+// Vendor ID, Physical Function Device ID, Virtual Function Device ID
+var NicIdMap = []string{
+	"8086 1583 154c",
```

Comment:
We may need to remove the NIC models which we don't officially support. Since this PR will also make the webhook to use the same list for admission.

Reply:
removed